src/api/telemetry/TelemetryAPI.js

```diff
@@ -166,25 +196,27 @@ export default class TelemetryAPI {
     }
 
     /**
-     * @private
-     * Though used in TelemetryCollection as well
+     * @param {TelemetryRequestOptions} options options for the telemetry request
+     * @returns {TelemetryRequestOptions} the options, with defaults filled in
      */
-    standardizeRequestOptions(options) {
-        if (!Object.prototype.hasOwnProperty.call(options, 'start')) {
+    standardizeRequestOptions(options = {}) {
+        if (!Object.hasOwn(options, 'start')) {
             options.start = this.openmct.time.bounds().start;
         }
 
-        if (!Object.prototype.hasOwnProperty.call(options, 'end')) {
+        if (!Object.hasOwn(options, 'end')) {
             options.end = this.openmct.time.bounds().end;
         }
 
-        if (!Object.prototype.hasOwnProperty.call(options, 'domain')) {
+        if (!Object.hasOwn(options, 'domain')) {
             options.domain = this.openmct.time.timeSystem().key;
         }
 
-        if (!Object.prototype.hasOwnProperty.call(options, 'timeContext')) {
+        if (!Object.hasOwn(options, 'timeContext')) {
             options.timeContext = this.openmct.time;
```

Comment:
I have reservations about this assignment. I don't think we should do this as there might be consumers downstream who might be conditionally using code if timeContext is present vs absent.
Please confirm with @akhenry if this is OK.

Reply:
I'm fine either way. @ozyx suggested putting it in `options` for consistency, but I understand if downstream things are relying on it we don't want to break them. What say you @akhenry

Reply:
@shefalijoshi Good point. Combined with https://github.com/nasa/openmct/issues/6493 this would remove the ability to make requests with arbitrary bounds, which we don't want to do.

On the other hand, defining a default context would be consistent with the other options here, and potentially removes some boilerplate.

@scottbell Perhaps we should reconsider our approach to https://github.com/nasa/openmct/issues/6493? We could instead always use bounds, and instead default them to the given time context if bounds are not explicitly provided?

Reply:
@akhenry Ok, just to summarize that we'll:

1. Add a default `timeContext` if one isn't provided
2. Give primacy to explicitly passed bounds.
3. Add explicitly passed bounds to a passed `timeContext` (if they aren't provided in the passed `timeContext`).

Reply:
@akhenry @shefalijoshi I've modified the code to have bounds default to a time context's bounds if supplied. Otherwise we use the default time context for start/end, and define a default time context.